editor/property_selector.cpp

```diff
@@ -348,10 +348,28 @@ void PropertySelector::_confirmed() {
 	if (!ti) {
 		return;
 	}
-	emit_signal(SNAME("selected"), ti->get_metadata(0));
+	if (search_options->get_select_mode() == search_options->SELECT_SINGLE) {
```

Comment:
I thought about using a switch statement, but I think it would just take up more space and considering there is only two/ three states if you count SELECT_ROW I'm not sure if it's worth it.